service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/azure/managedIdentity/CreateFederatedIdentityStep.java

```diff
@@ -157,12 +156,11 @@ private void createK8sServiceAccount(CoreV1Api aksApi, String uamiClientId) thro
                     .name(ksaName)
                     .namespace(k8sNamespace));
 
-    aksApi.createNamespacedServiceAccount(k8sNamespace, k8sServiceAccount, null, null, null, null);
+    aksApi.createNamespacedServiceAccount(k8sNamespace, k8sServiceAccount).execute();
   }
 
   private void deleteK8sServiceAccount(CoreV1Api aksApi, String k8sNamespace) throws ApiException {
-    aksApi.deleteNamespacedServiceAccount(
-        ksaName, k8sNamespace, null, null, null, null, null, new V1DeleteOptions());
+    aksApi.deleteNamespacedServiceAccount(ksaName, k8sNamespace).execute();
```

Comment:
You lost the deleteOptions here.

Reply:
Example: https://github.com/DataBiosphere/terra-landing-zone-service/pull/423/files#diff-dc845e0597a636315d1033f7e01368dfcc4af46bf7d9e7f6b4ba21a3bec5281dR102

Reply:
Thanks for the example.

Reply:
I can add it back in, but do you know what the purpose of an empty `V1DeleteOptions()` object is?

Reply:
Ha, no I do not! I was just trying to make sure all the original arguments still were maintained. Since we weren't passing null for it, I naively assumed it was somehow important.

Reply:
All good!  I think even if it's not buying us anything, there's value in maintaining the original arguments within the scope of this upgrade, I added it back.